src/components/forms/FileInput/FileInput.tsx

```diff
@@ -12,164 +12,186 @@ interface FileInputProps {
-            {previewHeaderText}{' '}
-            <span className="usa-file-input__choose">
-              Change file{filePreviews.length > 1 && 's'}
-            </span>
-          </div>
-        )}
+        data-testid="file-input"
+        className={fileInputClasses}
+        aria-disabled={disabled}>
         <div
-          data-testid="file-input-instructions"
-          className={instructionClasses}
-          aria-hidden="true">
-          {!hideDragText && (
-            <span className="usa-file-input__drag-text">{dragText}</span>
+          data-testid="file-input-droptarget"
+          className={targetClasses}
+          onDragOver={handleDragOver}
+          onDragLeave={handleDragLeave}
+          onDrop={handleDrop}>
+          {filePreviews.length > 0 && (
+            <div
+              data-testid="file-input-preview-heading"
+              className="usa-file-input__preview-heading">
+              {previewHeaderText}{' '}
+              <span className="usa-file-input__choose">
+                Change file{filePreviews.length > 1 && 's'}
+              </span>
+            </div>
           )}
-          <span className="usa-file-input__choose">choose from folder</span>
-        </div>
-        {filePreviews}
-        <div data-testid="file-input-box" className="usa-file-input__box"></div>
-        {showError && (
           <div
-            data-testid="file-input-error"
-            className="usa-file-input__accepted-files-message">
-            This is not a valid file type.
+            data-testid="file-input-instructions"
+            className={instructionClasses}
+            aria-hidden="true">
+            {!hideDragText && (
+              <span className="usa-file-input__drag-text">{dragText}</span>
+            )}
+            <span className="usa-file-input__choose">choose from folder</span>
           </div>
-        )}
-        <input
-          {...inputProps}
-          ref={inputRef}
-          type="file"
-          data-testid="file-input-input"
-          name={name}
-          id={id}
-          className="usa-file-input__input"
-          disabled={disabled}
-          onChange={handleChange}
-          multiple={multiple}
-          accept={accept}
-        />
+          {filePreviews}
+          <div
+            data-testid="file-input-box"
+            className="usa-file-input__box"></div>
+          {showError && (
+            <div
+              data-testid="file-input-error"
+              className="usa-file-input__accepted-files-message">
+              This is not a valid file type.
+            </div>
+          )}
+          <input
+            {...inputProps}
+            ref={internalRef}
+            type="file"
+            data-testid="file-input-input"
+            name={name}
+            id={id}
+            className="usa-file-input__input"
+            disabled={disabled}
+            onChange={handleChange}
+            multiple={multiple}
+            accept={accept}
```

Comment:
Getting a type error here  for `enterKeyHint` being a missing prop as soon I added the new FileInput to the application. 

Adding that prop with the value of `enter` (default) resolved the issue. It sounded [similar to this issue](https://github.com/pluralsight/design-system/commit/eb920f5a7bdd6571d578a601a629d8d411fd07f0). 🤔 Not sure what to make of this.

Reply:
Oops this was an issue in our code. SMH excuse me.